_posts/2025-08-16-accessible-error-messages-in-forms.md

```diff
@@ -0,0 +1,158 @@
+
+## What do I really need to know?
+
+Handling errors starts with the questions you ask. Some questions are difficult to answer. Is it really essential to know whether someone is male or female? Do you only want to contact them by phone? Perhaps the user wants to provide a different answer or doesn't want to answer, can’t choose an option and then gives up.
+
+Only ask for information you truly need in order to process the request. The fewer questions you ask, the lower the barrier for filling out the form.
+
+Be very clear about why you need sensitive personal information, such as a citizen service number, race, religion or medical data. Check your local legislation if you are allowed to request such private information in an open online form.
+
+{: .note }
+In the Netherlands there are strict rules about [when to ask for a citizen service number](https://www.rijksoverheid.nl/onderwerpen/privacy-en-persoonsgegevens/vraag-en-antwoord/welke-organisaties-mogen-mijn-burgerservicenummer-bsn-gebruiken#:~:text=Alle%20overheidsorganisaties%20mogen%20gebruik%20maken%20van%20uw%20burgerservicenummer%20(BSN)).
+
+
+## How will I ask for this information?
+
+Will you design new custom input fields or use recognizable, familiar patterns? Please don’t reinvent the web for something as essential as form fields. As Heydon Pickering explains in his talk [Get Your Priorities Straight](https://www.youtube.com/watch?v=ediHVy0869c): “Real people aren't looking to be delighted. People want to get the task done and get on with their lives".
+
+{: .example }
+For example: users recognize radio buttons as circles and checkboxes as squares. They know from experience they can choose one option from the radio buttons and multiple options from the checkboxes. Also, the [keyboard interaction](https://webaim.org/techniques/keyboard/#testing) for radiobuttons and checkboxes differs from each other. 
+
+Some field types are better avoided due to poor usability, such as multi-selects. Using date pickers? Check the experience for keyboard- or screen reader users.
+
+Sometimes a simple solution works best for everyone — for example, KLM’s method for selecting a date with a text input for the day, a dropdown for the month, and another text input for the year:
+![Date selection with a text field for day, a select for month, and a text field for year](https://raw.githubusercontent.com/nl-design-system/documentatie/assets/blog_toegankelijker-foutmeldingen-date-picker.png)
+
+For a deep dive into asking for dates in forms, see the GOV.UK Design System pattern library: [Ask users for Dates](https://design-system.service.gov.uk/patterns/dates/).
+
+Don't force the user to enter information twice (like an email address) and remember in a multistep form entries given in previous steps.
```

Comment:
rephrase:

"Don't force the user to enter information (like an email address) more than once, or expect them to remember entries entered on previous screens, like in a multistep form."